### ISSUE TITLE
Compatibility issue with NVDA 2019.2.1

### ISSUE THREAD

**Issue by CyrilleB79** [ABuffEr/columnsReview#31]
### STR
* Start NVDA 2019.2.1 and install the add-on
* Open Windows Explorer and create an new folder
* When the new folder is selected, press Enter to go in this folder

### Actual behaviour
An error is logged and nothing is reported

### Expected behaviour
No error.

### Possible solutions
* Make the add-on compatible only with 2019.3 onwards and provide a download link for the last working version on Python 2 for NVDA 2017.3 to 2019.2.1.
* Or fix the encoding issue and maybe other ones with Python 2
I have no preference.

### Context
Testing other add-ons in my config on 2019.2.1. Having errors in the log is disturbing.

**Comment by ABuffEr** [comment]
Hi @CyrilleB79 ,
are you talking about stable version ot the add-on?
If yes, can you provide the log?
Thanks!

**Comment by CyrilleB79** [comment]
Sorry, my report was missing some information:

The version is 5.0 (stable).

Here is the error in the log:
```
ERROR - eventHandler.executeEvent (21:40:12.545):
error executing event: gainFocus on <NVDAObjects.Dynamic_CRList64UIAWindowNVDAObject object at 0x05517650> with extra args of {}
Traceback (most recent call last):
  File "eventHandler.pyo", line 155, in executeEvent
  File "eventHandler.pyo", line 92, in __init__
  File "eventHandler.pyo", line 100, in next
  File "C:\Users\Cyrille\AppData\Roaming\nvda\addons\columnsReview\globalPlugins\columnsReview\__init__.py", line 177, in event_gainFocus
  File "eventHandler.pyo", line 100, in next
  File "appModules\explorer.pyo", line 334, in event_gainFocus
  File "eventHandler.pyo", line 100, in next
  File "C:\Users\Cyrille\AppData\Roaming\nvda\addons\columnsReview\globalPlugins\columnsReview\__init__.py", line 664, in event_gainFocus
  File "C:\Users\Cyrille\AppData\Roaming\nvda\addons\columnsReview\globalPlugins\columnsReview\__init__.py", line 653, in reportEmpty
UnicodeEncodeError: 'ascii' codec can't encode character u'\xe9' in position 2: ordinal not in range(128)

```

**Comment by CyrilleB79** [comment]
Hi @ABuffEr

No this new build did not solve anything.

But I have fixed the issue in #32.

Probably you have not encountered the error described in this issue on your side because it only happens if the translation of "0 items" contains non-ASCII characters, which is not the case in English or in Italian.

**Comment by ABuffEr** [comment]
Thanks, I didn't knew your solution, I'm going to accept #32
